packages/@ember/string/index.ts

```diff
@@ -399,7 +411,7 @@ if (ENV.EXTEND_PROTOTYPES.String) {
       configurable: true,
       enumerable: false,
       writeable: true,
-      value: function() {
+      value: function () {
```

Comment:
Spacing seems odd here, and is failing the build

Reply:
*shakes fist at VS Code*